src/main/kotlin/org/taktik/icure/asyncdao/impl/CodeDAOImpl.kt

```diff
@@ -66,6 +66,8 @@ class CodeDAOImpl(
 	@Qualifier("asyncCacheManager") asyncCacheManager: AsyncCacheManager
 ) : CachedDAOImpl<Code>(Code::class.java, couchDbProperties, couchDbDispatcher, idGenerator, asyncCacheManager), CodeDAO {
 
+	val SMALLEST_CHAR = "\u0000"
```

Comment:
You may use 
```
companion object {
		private const val couchDBSmallestChar = "\u0000"
	} 
```
Generally, when we have constants, we use companion object in Kotlin so that they are put outside of the class.